normalizeAccountData.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const DEFAULT_ACCOUNT_DATA = require('./default_account_data');
+const isValidAccountDataSectionValue = require('./isValidAccountDataSectionValue');
+
+module.exports = function normalizeAccountData(accountData) {
+  const normalizedAccountData = {};
+  for (const section in DEFAULT_ACCOUNT_DATA){
+    normalizedAccountData[section] = {};
+    for (const key in DEFAULT_ACCOUNT_DATA[section]){
+      let newValue = accountData && accountData[section] && accountData[section][key];
```

Comment:
If there is no accountData shouldn't we handle that at the top of this function?
```
module.exports = function normalizeAccountData(accountData) {
   const normalizedAccountData = {}
   if (!accountData) return normalizedAccountData
...
}
```

Reply:
nope, its okay if you give it nothing